lib/puma/launcher.rb

```diff
@@ -58,6 +58,28 @@ def initialize(conf, launcher_args={})
 
       @config.load
 
+      if @config.options[:bind_to_activated_sockets]
```

Comment:
IMO all of this should live in `Binder`, not in the `Launcher`

Reply:
I just pushed another commit. Is that roughly what you were thinking about?

Reply:
Yup!